app/controllers/experience_levels_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ExperienceLevelsController < ApplicationController
+  before_action :check_user_is_admin
+  def index
+    @experience_levels = ExperienceLevel.all
+  end
+
+  def new
+    @experience_level = ExperienceLevel.new
+  end
+
+  def create
+    @experience_level = ExperienceLevel.new(experience_level_params)
+    if @experience_level.save
+      redirect_to experience_levels_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @experience_level = ExperienceLevel.find(params[:id])
+  end
+
+  def update
+    @experience_level = ExperienceLevel.find(params[:id])
+    if @experience_level.update(experience_level_params)
+      redirect_to experience_levels_path
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def active
+    @experience_level = ExperienceLevel.find(params[:id])
```

Comment:
A `before_action` would fit well here, so we did not need to `find` the object on every action.